### PR TITLE
fixes it showing big bold in the ghost spawner for a few things

### PR DESCRIPTION
**Pull request by boodaliboo**
pm me if theirs more spawners with this
#### Changelog

:cl:  
bugfix: fixed  it showing big bold for a few ghost spawners
/:cl:

### CODE REVIEW

File: _maps/RandomRuins/SpaceRuins/spacebar.dmm [hunk 1]
```diff
@@ -509,7 +509,7 @@
 /area/ruin/powered/spacebar)
 "bC" = (
 /obj/effect/mob_spawn/human/bartender/alive{
-	flavour_text = "<span class='big bold'>You are a space bartender!</span><b> Time to mix drinks and change lives. Do not leave your post under any circumstances!! </b>";
+	flavour_text = "You are a space bartender!</span><b> Time to mix drinks and change lives. Do not leave your post under any circumstances!!";
```
Comment (Bibby0110) [inline _maps/RandomRuins/SpaceRuins/spacebar.dmm]: No introductory span, yet a closing one
Opening bold tag, no closing.

File: code/game/objects/structures/ghost_role_spawners.dm [hunk 2]
```diff
@@ -262,9 +262,9 @@
 	name = "broken rejuvenation pod"
 	desc = "A small sleeper typically used to instantly restore minor wounds. This one seems broken, and its occupant is comatose."
 	mob_name = "a translocated vet"
-	flavour_text = "<span class='big bold'>What...?</span><b> Where are you? Where are the others? This is still the animal hospital - you should know, you've been an intern here for weeks - but \
+	flavour_text = "<span class='What...?</span><b> Where are you? Where are the others? This is still the animal hospital - you should know, you've been an intern here for weeks - but \
```
Comment (Bibby0110) [inline code/game/objects/structures/ghost_role_spawners.dm]: Empty class statement, no closing arrow thing.

File: code/game/objects/structures/ghost_role_spawners.dm [hunk 3]
```diff
@@ -262,9 +262,9 @@
...
 	everyone's gone. One of the cats scratched you just a few minutes ago. That's why you were in the pod - to heal the scratch. The scabs are still fresh; you see them right now. So where is \
-	everyone? Where did they go? What happened to the hospital? And is that <i>smoke</i> you smell? You need to find someone else. Maybe they can tell you what happened.</b>"
+	everyone? Where did they go? What happened to the hospital? And is that <i>smoke</i> you smell? You need to find someone else. Maybe they can tell you what happened."
```
Comment (Bibby0110) [inline code/game/objects/structures/ghost_role_spawners.dm:267]: No closing bold tag, yet an open one on line 265

**Review by Bibby0110 — CHANGES_REQUESTED** [review]
Why are you even changing this?
They're big for emphasis

File: _maps/RandomRuins/SpaceRuins/spacebar.dmm (hunk 1 above)
Comment (JamieD1) [inline _maps/RandomRuins/SpaceRuins/spacebar.dmm]: Either create a new span open tag or remove it. Also remove the bold tag or close it.

File: code/game/objects/structures/ghost_role_spawners.dm (hunk 3 above)
Comment (JamieD1) [inline code/game/objects/structures/ghost_role_spawners.dm:267]: If you are removing the bold close tag then remove the b open tag.

**Review by JamieD1 — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/ghost_role_spawners.dm (hunk 2 above)
Comment (JamieD1) [inline code/game/objects/structures/ghost_role_spawners.dm]: <span class='What...?</span>

This needs to be completely redone..

**Review by alexkar598 — CHANGES_REQUESTED** [review]
I can understand why you would want to remove the big part of it, but what does the bold do bad?

Either come up with a reason to remove it or close this PR

**Comment by boodaliboo** [comment]
i dont feel like explaining to alex why me removing it is needed

**Pull request closed without merging**